modules/developers-guide/pages/graphql-using.adoc

```diff
@@ -415,7 +415,7 @@ include::example$result/gql_dropTableIfExists.result[]
 // end::DropAdvTable[]
```

Comment:
Should this file be renamed "graphql.adoc"?

Reply:
No. There is graphql.adoc (overview, general info), graphql-using.adoc (cql-first), and graphql-first-using.adoc (schema-first).